pywinauto/unittests/test_uia_element_info.py

```diff
@@ -62,6 +62,22 @@ def testChildren(self):
             """Test whether a list of only immediate children of the element is equal"""
             self.assertEqual(len(self.ctrl.children()), 5)
 
+        def testDefaultDepthDescendants(self):
+            """Test whether a list of descendants with default depth of the element is equal"""
+            self.assertEqual(len(self.ctrl.descendants()), 69)
```

Comment:
I'm not big fan of such hard-coded numbers. The amount of controls can be changed later. Can we think  about something more dynamic ?